app/components/traffic-measure/index.js

```diff
@@ -189,33 +237,20 @@ export default class TrafficMeasureIndexComponent extends Component {
     this.preview = this.template.value;
 
     for (let i = 0; i < this.mappings.length; i++) {
-      const e = this.mappings[i];
+      const e = this.mappings.objectAt(i);
```

Comment:
in general prefer the following construct over a regular for if you don't need the index.
```
for (const mapping of this.mappings) {

}
```

Also make sure to use proper naming, `e` isn't very descriptive.

Reply:
oki